Gopkg.lock

```diff
@@ -187,7 +187,10 @@
 
 [[projects]]
   name = "github.com/prometheus/client_golang"
-  packages = ["prometheus"]
+  packages = [
+    "prometheus",
+    "prometheus/promhttp"
+  ]
```

Comment:
@lordofthejars With humanly observation this change look fine to me.  'dep' tool auto-generate this .lock file.  Please try using it to avoid any future human mistake.  You can refer https://gist.github.com/subfuzion/12342599e26f5094e4e2d08e9d4ad50d or we can have bluejeans session to do it together.